app/main.py

```
@@ -412,17 +450,51 @@ def fetch_and_update_metrics():
 
+def fetch_and_update_status_metrics():
+    """
+    Continuously poll TeamCity and refresh only BUILD_STATUS_GAUGE metrics.
+    This function runs more frequently than the full metrics update.
+    """
+    logging.info("Starting status metrics update thread")
+
+    while True:
+        try:
+            update_build_status_metrics()
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)